src/PrestaShopBundle/Form/Admin/Product/ProductOptions.php

```diff
@@ -204,7 +204,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder->add('attachment_product', 'PrestaShopBundle\Form\Admin\Product\ProductAttachement', array(
             'required' => false,
             'label' => $this->translator->trans('Attachment', [], 'Admin.Catalog.Feature'),
-            'attr' => ['data-action' => $this->router->generate('admin_product_attachement_add_action')]
+            'attr' => ['data-action' => $this->router->generate('admin_product_attachement_add_action', array('idProduct' => 1))]
```

Comment:
I think yes it's better, but, we have to change all (because, all regex works with \d) so we have to change all, but i'm agree with you..